proto/common.proto

```diff
@@ -194,3 +194,11 @@ enum DistanceType {
   DISTANCE_TYPE_COSINE = 3;
   DISTANCE_TYPE_INNER_PRODUCT = 4;
 }
+
+enum ThrottleType {
```

Comment:
Type -> Object, as you are rate limit database objects

Reply:
This refers to the type of rate limit we are applying. Source, backfill, sink, dml these are all different types of rate limits.